docs/resources/yaml.md.erb

```diff
@@ -11,36 +11,36 @@ Use the `yaml` InSpec audit resource to test configuration data in a Yaml file.
 
 ## Syntax
 
-A `yaml` resource block declares the configuration data to be tested. Assume the following Yaml file:
+A `yaml` resource block declares the configuration data to be tested. Assume the following Yaml file called 'filename.yml':
 
-    name: foo
-    array:
-      - zero
-      - one
+    name: myservice
+    hosts:
+      - ip1.host.com
+      - ip2.host.com
 
```

Comment:
The purpose of this example to is set up an array and then demonstrate how to query the array. This is one of the places for using the conventions `foo, bar, baz`, `tmp\example`, and `example.com`.